datasets/norne/norne.py

```diff
@@ -0,0 +1,206 @@
+# coding=utf-8
+# Copyright 2021 National Library of Norway.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""NorNE: Annotating Named Entities for Norwegian."""
+
+try:
+    import conllu
+except ImportError:
+    import sys
+
+    print("Please, 'pip install conllu' to use the NorNE dataset", file=sys.stderr)
+
```

Comment:
```suggestion
import conllu
```

You can simply do `import conllu`.
The `datasets` library checks the imports before loading the script, and if `conllu` is not installed it's prompted to the user to install it :)

Reply:
The reason I added that is so the dataset could be explored using the datasets explorer web-based UI. But just removed it.

Reply:
We'll install conllu in the viewer for the dataset to be explorable